client/README.md

```diff
@@ -192,3 +192,1456 @@ For example:
 ## Deployment
 
 - Deployed on Netlify: https://cotripper.netlify.com/
+
+#### Seeding Heroku
```

Comment:
We should use [Django fixtures](https://docs.djangoproject.com/en/3.0/howto/initial-data/)